src/utils/postErrorToSlack.js

```diff
@@ -109,7 +102,7 @@ const postError = async (errorLog, context) => {
   }
 };
 
-const postErrorToSlack = async (error, info, reduxDump) => {
+const postErrorToSlack = async (error, reduxDump) => {
```

Comment:
rename `reduxDump` to `reduxState` and `error` to `errorObject` to be consistent with the names used for these everywhere else.

Reply:
renamed